packages/gitea/zarf.yaml

```diff
@@ -57,6 +65,12 @@ components:
           - gitea-values.yaml
     actions:
       onDeploy:
+        before:
+          - cmd: echo ${ZARF_VAR_GITEA_IMAGE_TAG%-rootless}
+            description: "Strip rootless from tag because helm will append it."
+            mute: true
+            setVariables:
+              - name: GITEA_IMAGE_TAG
```

Comment:
This probably shouldn't be here - we could rename `TAG` to `SERVER_VERSION` to make it clear this is the Gitea server version not a literal docker tag.  I am personally not a fan of these shim mutations (especially when it can be solved with better user messaging)